www/index.js

```diff
@@ -1,11 +1,14 @@
-var assert = require('nanoassert')
```

Comment:
What's the reason for this?

Reply:
Removing a package dependency that was just resolving to an if statement.

Reply:
That's on purpose :)

Reply:
Ok, but using this with the cordova.plugin syntax generates an error of "module nanoassert not found".

Reply:
Can you try with the `cordova plugin add --browserify ...` syntax and see if that works?

Reply:
I'll give it a try, but I think it's also a bit overheading adding a dependency for small purposes. I think your library is very useful, though.

Reply:
Just tried with browserify switch... nothing to do `Uncaught module nanoassert not found`

Reply:
I've also been searching a bit and seems that require() can't be used in www/*.js files